src/main/resources/default/taglib/t/blobImageSoftRefField.html.pasta

```diff
@@ -129,5 +149,37 @@
             }
             return '/' + path.substring(0, path.lastIndexOf('/') + 1);
         }
-    }
+
+        function isDisplayableImage(extension) {
```

Comment:
Would it be useful to make the extension toLowerCase, so that also possible capitalized extensions (e.g. .JPG) are considered.

Reply:
Good point. The previous logic expected an image and we are able to handle upper cased file extensions, thus uppercased extensions were accepted. I now added this functionality back in.